CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ _June 06, 2024_
 
 ## StreamChat
 ### ✅ Added
+- Add support for user blocking [#3223](https://github.com/GetStream/stream-chat-swift/pull/3223)
+
+### 🔄 Changed
```

Comment:
This is incorrectly added in 4.57.0, instead of upcoming

Reply:
That's weird. I definitely did move it up to the upcoming one. Fixed again now.